moduleframework/mtf_scheduler.py

```diff
@@ -194,9 +196,14 @@ class AvocadoStart(object):
 
     def __init__(self, args, unknown):
         # choose between TESTS and ADDITIONAL ENVIRONMENT from options
+        if args.static_linters:
+            self.tests += glob.glob("{MTF_TOOLS}/{STATIC_LINTERS}/*.py".format(
+                MTF_TOOLS=metadata_common.MetadataLoaderMTF.MTF_LINTER_PATH,
+                STATIC_LINTERS=common.STATIC_LINTERS))
         if args.linter:
-            self.tests += glob.glob("{MTF_TOOLS}/*.py".format(
-                MTF_TOOLS=metadata_common.MetadataLoaderMTF.MTF_LINTER_PATH))
+            self.tests += glob.glob("{MTF_TOOLS}/{GENERIC_TEST}/*.py".format(
```

Comment:
this change cause regression (now args.linter schedule all generic tests) with your change, just part of them will be sheduled. Is that intentional? we can discuss it. I'm not against this idea, but we should think about it more.